packages/opentelemetry-plugin-http/test/utils/httpRequest.ts

```diff
@@ -52,10 +52,10 @@ function get(input: any, options?: any): GetResult {
         reject(err);
       });
     }
-    req =
-      options != null
-        ? http.get(input, options, onGetResponseCb)
-        : http.get(input, onGetResponseCb);
+    const isValid = options !== null && options !== undefined;
```

Comment:
What rule triggered this rewrite?

Reply:
!= null, it wants !== and it's fairly rare that we use != or ==

Reply:
`!=` can be useful as it matches both null and undefined. Maybe we should allow that?

Reply:
I edited the rule.